src/main/java/chopchop/logic/commands/StatsRecipeMostCookedCommand.java

```diff
@@ -22,7 +22,7 @@ public CommandResult execute(Model model, HistoryManager historyManager) {
 
     @Override
     public boolean equals(Object other) {
-        return other == this || (other instanceof FindRecipeCommand);
+        return other == this;
```

Comment:
shouldn't this check that it's `instanceof StatsRecipeMostCookedCommand` also? though idk who tf is out there calling `.equals()` on instances of Command...

Reply:
Wait I don't understand if it's just an instance of `StatsRecipeMostCookedCommand` and it got no attributes to compare. Then it means comparing this with any other `StatsRecipeMostCookedCommand` will result in true? This is prob a stupid question but yea Im just pikachu

Reply:
ya since the class don't have any attributes i suppose any `StatsRecipeMostCookedCommand` is equal to any other `StatsRecipeMostCookedCommand`

tbh i pikachu also